Extensions/NotificationService/NotificationService.swift

```diff
@@ -102,6 +102,11 @@ class SyncDataDisplay {
 
         switch message {
         case .commandReceived(let tab):
+            // The "old new Send Tab" format includes the message payload in the
+            // notification, while "new new Send Tab" supplies an `index` so that
+            // the payload can be fetched from the FxA server. Firefox for iOS
+            // only supports "old new Send Tab", which doesn't include the
+            // `flowID` in the message payload.
```

Comment:
@eoger I think this means we can only record telemetry for tabs sent through clients collection commands, not pushes or FxA commands...is that right? @mhammond recently added `streamID` that's unique per target (though I think our `flowID` will do that here?), but not for the clients collection.

I see there's also `Profile.sendItem` that calls `constellation.sendEventToDevice`—I guess I'll want to change that to send along a stream and flow ID, too, right?